language/bytecode_verifier/src/type_memory_safety.rs

```diff
@@ -830,19 +918,21 @@ impl<'a> TypeAndMemorySafetyAnalysis<'a> {
                 }
             }
 
-            // TODO: Handle type actuals for generics
-            Bytecode::MoveToSender(idx, _) => {
+            Bytecode::MoveToSender(idx, type_actuals_idx) => {
                 let struct_definition = self.module().struct_def_at(*idx);
                 if !StructDefinitionView::new(self.module(), struct_definition)
```

Comment:
Are we doing any additional checks here? Maybe that `type_actuals.len() == 0`? I remember thinking about putting that restriction on for the short term. @runtian-zhou ?

Reply:
Will do in signature checker.